assets/prototype/domain/eventEditor/datesList/dateCard/EditDateButton.js

```diff
@@ -0,0 +1,24 @@
+import { useContext } from '@wordpress/element';
+import { Button } from '@blueprintjs/core/lib/esm';
+import { DateTimeContext } from '../../../../infrastructure/services/contextProviders/DateTimeProvider';
+
+const EditDateButton = () => {
+	const { isOpen, setIsOpen } = useContext(DateTimeContext);
+	const onClick = () => setIsOpen(!isOpen);
+
+	return (
+		<div
+			style={{
+				top: '.5rem',
+				position: 'absolute',
+				right: '.5rem',
+				textAlign: 'right',
+				zIndex: '1',
+			}}
+		>
```

Comment:
we need to extract this layout logic because it totally limits where this button can be used.
ie: this layout logic belongs in the consumer that is positioning this button.

Reply:
passed `position` prop